### PR TITLE
deps: run npm audit fix

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,12 +49,12 @@
     "i18next-xhr-backend": "3.0.0",
     "jQuery-Impromptu": "github:trentrichardson/jQuery-Impromptu#v6.0.0",
     "jitsi-meet-logger": "github:jitsi/jitsi-meet-logger#5ec92357570dc8f0b7ffc1528820721c84c6af8b",
-    "jquery": "3.4.0",
+    "jquery": "3.5.1",
```
Comment (luixxiul) [inline package.json:52]: `Closes https://github.com/jitsi/jitsi-meet/pull/5629`

Comment (luixxiul) [reply package.json:52]: and `Closes https://github.com/jitsi/jitsi-meet/pull/6476`

File: package.json [hunk 2]
```diff
@@ -49,12 +49,12 @@
...
     "jquery-contextmenu": "2.4.5",
     "jquery-i18next": "1.2.1",
     "js-md5": "0.6.1",
     "js-utils": "github:jitsi/js-utils#cf11996bd866fdb47326c59a5d3bc24be17282d4",
-    "jsrsasign": "8.0.12",
+    "jsrsasign": "8.0.15",
```
Comment (luixxiul) [inline package.json:57]: note: cherry-picked from https://github.com/jitsi/jitsi-meet/pull/6120/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R57

File: package.json [hunk 3]
```diff
@@ -124,14 +124,14 @@
     "imports-loader": "0.7.1",
     "jetifier": "1.6.4",
     "metro-react-native-babel-preset": "0.56.0",
-    "node-sass": "4.13.1",
+    "node-sass": "4.14.1",
     "string-replace-loader": "2.1.1",
     "style-loader": "0.19.0",
     "unorm": "1.6.0",
-    "webpack": "4.27.1",
+    "webpack": "4.43.0",
```
Comment (luixxiul) [inline package.json:131]: This `closes #6120`. See: https://github.com/jitsi/jitsi-meet/pull/6120/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R130

**Comment by saghul** [comment]
@damencho I tried to repro some of the failures locally and I couldn't. Can you help me understand / get a better traceback?

**Comment by damencho** [comment]
Screenshot shows empty page and driver logs are empty ... so no errors ... that is so strange

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
@damencho This PR passes the tests now. I think the webpack issue might be related to the Node version. I’ll be looking into that this week, but we can land this already. PTAL when you get a chance.

**Pull request merged**